Scripts/Interfaces/IUsesToolTip.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace UnityEngine.Experimental.EditorVR
+{
+	/// <summary>
+	/// Provides access to the ability to show or hide a Tooltip
+	/// </summary>
+	public interface IUsesTooltip
```

Comment:
Not exactly. We add show/hide to rayEnter/Exit on the input module so that any UI with an ITooltip in the hierarchy will show its tooltip.